recipes/influxdb-cxx/all/test_package/test_package.cpp

```diff
@@ -0,0 +1,16 @@
+#include <InfluxDBFactory.h>
+#include <InfluxDBException.h>
+#include <iostream>
+
+int main()
+{
+    try {
+        auto influxdb = influxdb::InfluxDBFactory::Get("xyz://foobar");
```

Comment:
I've changed this to _not_ reference localhost - in the odd chance something is actually running.  Test packages in Conan center _must not_ make network requests to any resource whatsover.